packages/syft/tests/trials/skill_test.py

```diff
@@ -45,8 +46,10 @@ def test_trial_of_skill() -> None:
     assert get_padawans("R2Q4")["amdjed"] == "PASSED"
     assert get_padawans("R2Q4")["Osam"] == "PASSED"
     assert get_padawans("R2Q4")["Mikaela"] == "PASSED"
+    assert get_padawans("R2Q4")["jauhar"]  == "PASSED"
```

Comment:
TYPO: First letter is capital in the above dictionary, could you modify it?

Reply:
Done. Please have a look.